sphinx/cmd/quickstart.py

```diff
@@ -82,8 +82,7 @@
 # function to get input from terminal -- overridden by the test suite
 def term_input(prompt):
     # type: (unicode) -> unicode
-    print(prompt, end='')
-    return input('')
+    return input(prompt)
```

Comment:
I've found a problem on Windows.

![image](https://user-images.githubusercontent.com/151623/41181234-04dd28a8-6bac-11e8-9bb4-c129a3887d69.png)

```
C:\Users\taka>python
Python 3.6.2 (v3.6.2:5fd33b5, Jul  8 2017, 04:14:34) [MSC v.1900 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from sphinx.util.console import colorize
>>> colorize('yellow', 'Hello', True)
'\x01\x1b[33;01m\x02Hello\x01\x1b[39;49;00m\x02'
>>> colorize('yellow', 'Hello', False)
'\x1b[33;01mHello\x1b[39;49;00m'
>>> from sphinx.util.console import color_terminal
>>> color_terminal()
True
>>> input(colorize('yellow', 'Hello', False))
[33;01mHello[39;49;00m False
' False'
>>> input(colorize('yellow', 'Hello', True))
[33;01mHello[39;49;00m True
' True'
>>> print(colorize('yellow', 'Hello', False))
Hello
>>> print(colorize('yellow', 'Hello', True))
Hello
```

I guess `input()` doesn't handle escape sequence correctly.
Previous code (print + input) works fine.

Reply:
https://github.com/tk0miya/sphinx/commit/678cc3d50363e927ab904b61e9d7eaee04f4e965 separate `term_input` into `print` and `input` that shipped with Sphinx-1.5.3.

Before sphinx-1.5.3, Windows can't print escape sequence with `input` correctly too.
![image](https://user-images.githubusercontent.com/151623/41183215-6f4f2288-6bb4-11e8-875e-cdc65f54a5b7.png)

I think we should keep `print` + `input` code for windows environment.